Description: Fixes the compilation errors with Java 8
 Also explicitly sets compilation target to 8 since invoking default interface methods does not
 work for bytecode targeting > 8.
Author: Emmanuel Bourg
Forwarded: not-needed
--- a/cnf/includes/jdt.bnd
+++ b/cnf/includes/jdt.bnd
@@ -1,4 +1,4 @@
 -include: ${project}/.settings/org.eclipse.jdt.core.prefs
-javac.source 			= ${org.eclipse.jdt.core.compiler.source}
-javac.target 			= ${org.eclipse.jdt.core.compiler.codegen.targetPlatform}
-javac.compliance 		= ${org.eclipse.jdt.core.compiler.compliance}
+javac.source 			= 8
+javac.target 			= 8
+javac.compliance 		= 8
